### PR TITLE
Keven

### PR DESCRIPTION
**Pull request by kpmcdowellmo**
Added location and state of being to context resolution.
Tested context resolution strategy and database implementation.
Should be ready to hook into the Raspberry PI after adding server stuff.

### CODE REVIEW

File: prototypes/database/brandon_db_context_resolution.py [hunk 1]
```diff
@@ -0,0 +1,373 @@
+"""
```
Comment (brandonnorman) [inline prototypes/database/brandon_db_context_resolution.py]: Could go ahead and just delete this, as opposed to moving it.

File: prototypes/database/database_manager.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
+"""
```
Comment (brandonnorman) [inline prototypes/database/database_manager.py]: Also fine to just delete this.

Comment (kpmcdowellmo) [reply prototypes/database/database_manager.py]: I'm actually using database_manager in the main code now, but I will go ahead and delete this extra copy. I didn't realize it was there.

File: prototypes/database/brandon_db_context_resolution.py (hunk 1 above)
Comment (kpmcdowellmo) [reply prototypes/database/brandon_db_context_resolution.py]: Will do.

**Pull request merged**